aws/resource_aws_elb_test.go

```diff
@@ -1086,16 +1068,6 @@ func testAccCheckAWSELBDisappears(loadBalancer *elb.LoadBalancerDescription) res
 
 func testAccCheckAWSELBAttributes(conf *elb.LoadBalancerDescription) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
-		zones := []string{"us-west-2a", "us-west-2b", "us-west-2c"}
```

Comment:
Additional note here: These legacy API value checks within custom `TestCheckFunc` are generally discouraged in favor of checking the values directly in the Terraform state (e.g. `resource.TestCheckResourceAttr()`) and `ImportStateVerify` testing.

Reply:
:+1: